Scheduler/v6/maintel_fbs_sit_block_t87.yaml

```diff
@@ -0,0 +1,69 @@
+s3instance: cp
+auxtel:
```

Comment:
You can probably remove this auxtel session, but it is also harmless if you leave it.